enterprise/frontend/src/metabase-enterprise/sandboxes/components/EditSandboxingModal/EditSandboxingModal.tsx

```diff
@@ -397,8 +399,9 @@ const TargetName = ({ policy, policyTable, target }: TargetNameProps) => {
     ) {
       return (
         <span>
-          {c("{0} is a name of a variable being used by sandboxing")
-            .jt`${(<strong>{target[1][1]}</strong>)} variable`}
+          {c(
+            "{0} is a name of a variable being used by row and column security",
```

Comment:
```suggestion
            "{0} is a name of a variable",
```

The contextual string was too verbose to begin with. Let's simplify it